### DISCUSSION TITLE
Iniciar carreira Dev FRONT PJ em 2020

### DISCUSSION THREAD

**Discussion by Marilzon** [frontendbr/forum#1783] [Discussão]
Olá amigos, estou com muitas duvidas, pretendo **iniciar como PJ** e li vários artigos sobre o assunto, mas no final não fica um bom entendimento, para eu iniciar como PJ, logo:
 - Eu deveria contratar um _contator_ ou _empresa do ramo_ para tratar de assuntos burocráticos referente a impostos e declarações? 
 - Precisaria de Inscrição municipal
 - Onde eu me apoiaria ou que plataforma conseguiria ter orientação sobre o assunto?
 - Impostos iniciais e taxas?

Tenho bastante duvidas, o que vocês me indicam e para que lado devo caminhar e me direcionar sobre este assunto!?

**Comment by Marilzon** [comment]
@heliomsolivas Que legal, eu tinha informações de que Desenvolvedor não era permitido ser **MEI**, vou acessar este site e verificar todas informações importantes e atualizadas!

**Comment by felipefialho** [comment]
Opa @Marilzon recomendo fortemente que veja essas discussões: #28 #471
